test/indexing.jl

```diff
@@ -1822,6 +1822,19 @@ end
         @test_throws MethodError df[1, 2:3] = DataFrame([11 12])
         @test_throws MethodError df[1, [false, true, true]] = DataFrame([11 12])
     end
+
+    @testset "cornercase of view indexing" begin
+        df = DataFrame(reshape(1:12, 4, :))
+        dfr = df[1, 3:2]
+        for idx in [:x1, :x2, :x3, :x4]
+            @test_throws ArgumentError dfr[idx]
+        end
+
+        sdf = df[1:1, 3:2]
```

Comment:
Did you mean to do this?
```suggestion
        sdf = df[1:1, 3:2]
        dfr = view(df, 1, 3:2)
```